src/cursor/cur_backup.c

```diff
@@ -181,6 +181,8 @@ __backup_log_append(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, bool active)
 	if (conn->log) {
 		WT_ERR(__wt_log_get_all_files(
 		    session, &logfiles, &logcount, &cb->maxid, active));
+		F_SET(conn->log, WT_LOG_FORCE_NEWFILE);
```

Comment:
I'm thinking I should clear this flag on error, but wasn't sure if there were considerations. I imagine this thread is already holding a lock that prevents concurrent access to the `conn->log` object such that it's not possible to accidentally clear a different thread's setting of the flag.